### PR TITLE
LRQA-41711 Add BuildRunner object to be executed in CI

### PR DESCRIPTION
**Pull request by michaelhashimoto**
http://issues.liferay.com/browse/LRQA-41711

### CODE REVIEW

**Comment by michaelhashimoto** [comment]
ci:test:sf

**Comment by michaelhashimoto** [comment]
ci:test:relevant

File: modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/GitWorkingDirectoryFactory.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+	public static GitWorkingDirectory newGitWorkingDirectory(
+		String upstreamBranchName, File repositoryDir, String repositoryName) {
+
+		if (!repositoryDir.exists()) {
+			throw new RuntimeException(
+				"Directory path not found " + repositoryDir);
+		}
+
+		String key = repositoryName + "__" + upstreamBranchName;
```
Comment (pyoo47) [inline modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/GitWorkingDirectoryFactory.java]: @michaelhashimoto Did you mean to use "__" and not just "_"?

**Comment by pyoo47** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/61117

**Pull request closed without merging**